ci/tasks/publish.sh

```
@@ -0,0 +1,85 @@
+if [ "$FROM_BUCKET_NAME" == "$TO_BUCKET_NAME" ]; then
+  echo "Skipping upload since buckets are the same..."
+elif [ ${#COPY_KEYS_ARRAY[@]} -eq 0 ]; then
+  echo "Skipping upload since COPY_KEYS is empty..."
+else
+  for file in "${COPY_KEYS_ARRAY[@]}"; do
+    file="${file/\%s/$VERSION}"
+
+    echo "$file"
+
+    # occasionally this fails for unexpected reasons; retry a few times
+    for i in {1..4}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)